sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -150,25 +150,31 @@ class JDBCImport(
   private def readJDBCSourceInParallel(): DataFrame = {
 
     if (importConfig.splitBy.nonEmpty) {
+      val defaultString="defaultValue"
       val (lower, upper) = spark.read
         .jdbc(buildJdbcUrl, importConfig.boundsSql, jdbcParams)
-        .selectExpr("cast(min as long) min", "max")
-        .as[(Option[Long], Option[Long])]
+        .selectExpr("cast(min as string) min", "cast(max as string) max")
+        .as[(Option[String], Option[String])]
         .take(1)
-        .map { case (a, b) => (a.getOrElse(0L), b.getOrElse(0L)) }
+        .map { case (a, b) => (a.getOrElse(defaultString), b.getOrElse(defaultString)) }
         .head
 
-      spark.read
-        .jdbc(
-          buildJdbcUrl,
-          importConfig.jdbcQuery,
-          importConfig.splitColumn,
-          lower,
-          upper,
-          importConfig.chunks,
-          jdbcParams
-        )
-        .where(s"${importConfig.splitColumn} >= $lower and ${importConfig.splitColumn} <= $upper")
+      val jdbcUsername = dbutils.secrets.get(scope = databricksScope, key = "DB_USERNAME")
+      val jdbcPassword = dbutils.secrets.get(scope = databricksScope, key = "DB_PASSWORD")
+
+      spark.read.format("jdbc")
+        .option("url",buildJdbcUrl)
+        .option("dbtable",importConfig.jdbcQuery)
+        .option("user",jdbcUsername)
+        .option("password",jdbcPassword)
+        .option("partitionColumn",importConfig.splitColumn)
+        .option("lowerBound",lower)
```

Comment:
I think the string is not supported for lower and upper values, pls verify.

Reply:
I tested it on local, it was supporting.
 spark.read.jdbc(lower, upper) doesn't support strings(only take long values). But when we use spark.read.format("jdbc").options("lowerBound").options("upperBound") then string is supported.